go/vt/wrangler/testlib/init_shard_master_test.go

```diff
@@ -155,7 +184,8 @@ func TestInitMasterShardChecks(t *testing.T) {
 	// (master2 needs to run InitTablet with -force, as it is the second
 	// master in the same shard)
 	master2 := NewFakeTablet(t, wr, "cell1", 1, topodatapb.TabletType_MASTER, db, ForceInitTablet())
-	if err := wr.InitShardMaster(ctx, master2.Tablet.Keyspace, master2.Tablet.Shard, master2.Tablet.Alias, false /*force*/, 10*time.Second); err == nil || !strings.Contains(err.Error(), "is not the only master in the shard") {
+
+	if err := wr.InitShardMaster(ctx, master2.Tablet.Keyspace, master2.Tablet.Shard, master2.Tablet.Alias, false /*force*/, 10*time.Second); err == nil || !strings.Contains(err.Error(), "is not the shard master") {
```

Comment:
Why did this error change just now? I would have thought this test would have broken back when we made false masters demote themselves.

Reply:
ShardSync doesn't run until we start the tablet. So this test didn't change when that was implemented.
Now that we have changed InitTablet to not set MasterAlias on the shard, that value is nil, so the error has changed.

Reply:
I'm still worried this test is now no longer testing what it was meant to (detecting 2 masters in ISM). I think to keep the test an accurate simulation of 2 masters, we'd need to force the shard record to match one of the masters so ISM reaches the code that checks for 2 masters.

With that said, it may not be worth the effort to truly fix this test because 2 masters should not happen anymore anyway, and before long we want to do a whole overhaul of ISM. WDYT?